.github/workflows/nodejs-ci.yml

```diff
@@ -27,3 +27,8 @@ jobs:
     - run: npm install
     - run: npm run build --if-present
     - run: npm test
+    - run: npm run coverage
```

Comment:
may need to move this to a separate file...

Reply:
we'll see; let's try it. Looks like CI ran correctly but badge fail because of branch permission; it might succeed on main (let's see)